dashboard/app/views/levels/editors/_gamelab.html.haml

```diff
@@ -27,6 +27,10 @@
   = f.label :show_d_pad
   = boolean_check_box f, :show_d_pad
 
+.field
+  = f.label :expand_debugger, 'Expand Debugger by Default'
+  = boolean_check_box f, :expand_debugger
```

Comment:
It might seem silly right now, but I'd prefer a debugger editor partial (like the watchers_ui partial below) that gets used in both places so that we only have to modify one place to add debugger level options in the future.